docs/en/writing-running-appium/caps.md

```diff
@@ -280,3 +282,11 @@ Driver](/docs/en/drivers/ios-uiautomation.md).
 
 ### WinAppDriver Only
 (For WinAppDriver specific capabilities, please refer to the documentation on the [Appium Windows Driver repo](https://github.com/appium/appium-windows-driver#windowsdriver-specific-capabilities) itself.)
+
+### Flutter driver only 
```

Comment:
it is better to manage capabilities on their own repositories like https://github.com/appium/appium-xcuitest-driver#desired-capabilities , I believe